edx_solutions_api_integration/courses/tests.py

```diff
@@ -2662,8 +2685,8 @@ def test_course_navigation(self):
                 'vertical': unicode(self.content_child2.location),
                 'section': unicode(self.course_content2.location),
                 'course_key': unicode(self.course.id),
-                'final_target_id': unicode(self.content_subchild.location),
-                'position': '1',
+                'final_target_id': unicode(self.content_child.location),
```

Comment:
reason for changing `content_subchild` to `content_child`

Reply:
Getting content_child as final_target_id now.